pcdsdevices/cvmi_motion.py

```diff
@@ -41,6 +46,15 @@ class CVMI(BaseInterface, GroupDevice):
 
     sample_paddle = Cpt(BeckhoffAxis, ':MMS:07', kind='normal')
 
+    # LEDs
+    led1_name = Cpt(EpicsSignal, ':LED:01:NAME', kind='normal')
+    led1_pct = Cpt(EpicsSignal, ':LED:01:ILL:PCT', kind='normal')
+    led1_pwr = Cpt(EpicsSignalRO, ':LED:01:PWR', kind='normal')
+
+    led2_name = Cpt(EpicsSignal, ':LED:02:NAME', kind='normal')
```

Comment:
You might also notice, though, that you're repeating the lines of code a bit here.

Would you consider instead:
```python
class CvmiLED:
    name = Cpt(PytmcSignal, ":NAME", io="io")
    percent = Cpt(...)
    power = Cpt(...)

class CVMI:  # note that I'm skipping a lot of stuff here just to show the example
    led1 = Cpt(CvmiLED, ":LED:01")
    led2 = Cpt(CvmiLED, ":LED:02")
```

Reply:
How would Epics then know to put the ":NAME" at the end of the ":LED:01," considering the happi entry looks at the CVMI class?